assemblyline_core/expiry/run_expiry.py

```diff
@@ -145,6 +145,7 @@ def try_run(self):
             except Exception as e:
                 self.log.exception(str(e))
 
+            self.heartbeat()
             time.sleep(self.config.core.expiry.sleep_time)
```

Comment:
Expiry sleep time + the query processing time could be bigger then the arbitrary 30 sec set in the docker health check?

Reply:
Fair enough, expiry can take a long time, this probably isn't a good way to monitor it